tests/gui/pages/main.spec.js

```diff
@@ -26,6 +27,14 @@ const availableDrives = require('../../../lib/shared/models/available-drives')
 const selectionState = require('../../../lib/shared/models/selection-state')
 require('angular-mocks')
 
+// Mock HTML requires by reading from the file-system
+// eslint-disable-next-line node/no-deprecated-api
+require.extensions['.html'] = (module, filename) => {
```

Comment:
Out of curiosity, why is this snippet needed in `main.spec.js` but not in `finish.spec.js` or `settings.spec.js`?